app/scripts/models/location.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Helper Module
+ */
+
+define(['googlemaps!', 'gmaps', 'Helpers'], function(googleMaps, GMaps, Helpers) {
+  google.maps === googleMaps;
+
+  var Location = {};
+
+  Location = {
+
+  	getGeoCoordinates: function(callback) {
+  		navigator.geolocation.getCurrentPosition(function(position) {
```

Comment:
Why don't you handle the error? The first param is success callback, next is error callback.